### PR TITLE
Random fixes as I come across things

### PR DESCRIPTION
**Pull request by eromangaka**
Fixes lots of text positioning and random translations throught, im just going through the game now to see whats still not been touched.

### CODE REVIEW

**Comment by eromangaka** [comment]
2nd commit fixes a dumb search and replace I did.

File: www/data/CommonEvents.json [hunk 1]
```diff
@@ -2311825,7 +2311825,7 @@
                 "code": 401,
                 "indent": 0,
                 "parameters": [
-                    "\\>\\@Kana_en\\c[0]\\< \\>> I'm always sorry, \\< \\> for doing so\nmuch for me... \\<"
+                    "\\>\\@Kana_en\\c[0]\\< \\>> I'm always sorry,\\< \\>for have you do so\nmuch for me... \\<"
```
Comment (dazedanon) [inline www/data/CommonEvents.json:2311828]: 'having you do'

**Review by dazedanon — APPROVED** [review]
Nice work! 👍

File: www/data/CommonEvents.json (hunk 1 above)
Comment (eromangaka) [reply www/data/CommonEvents.json:2311828]: Yep, thanks for catching (and fixing) it.

**Pull request merged**